src/configure/configure.ts

```diff
@@ -108,7 +108,8 @@ class Orchestrator {
         await this.getInputs(node);
         if (this.continueOrchestration) {
             if (extensionVariables.remoteConfigurerEnabled === true && this.inputs.sourceRepository.repositoryProvider === RepositoryProvider.Github &&
-                (this.inputs.targetResource.resource.type === TargetResourceType.AKS || (this.inputs.pipelineConfiguration.template.language === "Node" && this.inputs.targetResource.resource.type === TargetResourceType.WebApp)) && !!this.inputs.sourceRepository.remoteUrl) {
+                (this.inputs.targetResource.resource.type === TargetResourceType.AKS || (this.inputs.pipelineConfiguration.template.language === "Node" && this.inputs.targetResource.resource.type === TargetResourceType.WebApp)
```

Comment:
please move this out to a separate function and make it easy to understand. We can call the function `DoesLanguageAndTargetSupportRemoteProvisioning()`